main.py

```diff
@@ -831,6 +834,14 @@
             id='trinket_1',
             options=[
                 {'label': 'Empty', 'value': 'none'},
+                {
+                    'label': 'Death\'s Verdict (H)',
+                    'value': 'Deaths Verdict heroic',
```

Comment:
Would keep the same style convention here for the value field as for the other trinkets just for consistency (no caps, underscore instead of spaces)